docs/post/memcards.md

```diff
@@ -0,0 +1,124 @@
+
+## Using MyMC
+
+### Prerequisites
+
+[The Windows Visual C++ 2008 Runtime (x86) is required](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist#visual-studio-2008-vc-90-sp1-no-longer-supported)
+
+:::info
+The x64 version will not work; you must install the x86 version if you do not have it already.
+:::
+
+### Download MyMC
+
+MyMC is a portable program and does not "install". [You can find the original MyMC here](http://www.csclub.uwaterloo.ca:11068/mymc/)
+
+Download the `mymc-alpha-2.7.zip` file and extract it to a folder of your choice. Launch MyMC by running `mymc-gui.exe`.
+
+## Using MyMC++
+
+### Prerequisites
+
+Windows and MacOS users will need to install the [latest version of Python from here.](https://www.python.org/downloads/)
+
+:::caution
+While going through the installer, make sure to check the box to add Python to your PATH. You may need to reboot your PC after installing to ensure your PATH variable is set properly; more on this later.
+
+<Image src={require("./img/memcard_python_path.webp").default} />
+:::
+
+:::tip
+Most Linux distributions come bundled with Python 3 already installed. If your distribution does not include Python 3, you should try first to install it using your distribution's package manager.
+
+It is also recommended that you use `pipx` to install MyMC++ as recent Python version have implemented externally managed environment, which forces you to first manually setup a virtual environment before you can start installing MyMC++. `pipx` will handle all of the process automatically.
+:::
+
+### Download MyMC++
+
+Once Python 3 is installed, you can use its package management command `pip` to automatically download and install MyMC++ for you. Open a command shell and run the following command:
+
+:::caution
+When running this command, do not run the `python` command or enter this into a Python shell. Enter this command directly into your operating system's command shell. For Windows users, enter this directly into Command Prompt or Powershell. For Linux and MacOS users, enter this directly into your terminal.
+:::
+
+```sh
+pip install mymcplusplus[gui]
+```
+
+If the above command fails saying that `pip` is an unrecognized command, you have not properly added Python to your PATH variable. Try rebooting first. If the command still fails, look up a guide for how to add Python to your operating system's PATH variable.
+
+Once `pip` finishes running, you can then run the following command from your operating system's command shell to launch MyMC++:
+
+Windows users can optionally type this command into their start menu, and Windows will suggest launching the program there. On any operating system, you can also create a shortcut and set this command as the target. Either of these methods will avoid needing to open a command shell to launch MyMC++.
```

Comment:
This comment should come after the command; the line above ends with a colon hinting that the command example is coming, so this comment should wait until after the command is stated.